FrontEnd/src/components/ProfilePage/FormComponents/FormFields/PasswordField.js

```diff
@@ -0,0 +1,65 @@
+import { useState } from 'react';
+import { PropTypes } from 'prop-types';
+import EyeInvisible from '../../../authorization/EyeInvisible';
+import EyeVisible from '../../../authorization/EyeVisible';
+import css from './PasswordField.module.css';
+
+const PasswordField = (props) => {
+    const [showPassword, setShowPassword] = useState(false);
+
+    const togglePassword = () => {
+        setShowPassword(!showPassword);
+    };
+
+    return (
+        <div className={css['password-field__item']}>
+            <div className={css['password-field__label-wrapper']}>
+                {props.requredField &&
+                <span>
+                    *
+                </span>}
+                <label
+                    htmlFor={props.inputId}
+                >
+                    {props.label}
+                </label>
+            </div>
+            <div className={css['password-field__password']}>
+                <div className={css['password-field__password__wrapper']}>
+                    <input
+                        id={props.inputId}
+                        name={props.name}
+                        type={showPassword ? 'text' : 'password'}
+                        value={props.value}
+                        onChange={props.updateHandler}
+                        placeholder={props.label}
+                        required={props.requredField}
+                    />
+                </div>
+                <span
+                    className={css['password-visibility']}
+                    onClick={togglePassword}
+                >
```

Comment:
The `toggle` button for showing/hiding the password doesn’t have an accessible label or an `aria-label` attribute. Adding accessible labels should be useful for users who rely on screen readers
e.g.
```
<span className={css['password-visibility']} onClick={togglePassword} aria-label={showPassword ? "Hide password" : "Show password"}>
   {!showPassword ? <EyeInvisible /> : <EyeVisible />}
</span>
```

Reply:
Thank you, will be done.